libminifi/CMakeLists.txt

```diff
@@ -48,6 +49,19 @@ file(GLOB SPD_SOURCES "../include/spdlog/*")
 add_library(spdlog STATIC ${SPD_SOURCES})
 add_library(minifi STATIC ${SOURCES})
 
+# Include Boost System
+find_package(Boost COMPONENTS system REQUIRED)
```

Comment:
I assume this was included for civet? all uses of pointer management in libminifi appeared to be std::